Casks/w/warp-preview.rb

```diff
@@ -0,0 +1,27 @@
+cask "warp-preview" do
```

Comment:
```suggestion
cask "warp@preview" do
```

The token should be `warp@preview` (see [documentation](https://docs.brew.sh/Cask-Cookbook#casks-pinned-to-development-channels)). Please also update the filename to match.

Reply:
Fixed